src/pipx/commands/run.py

```diff
@@ -319,41 +326,45 @@ def _http_get_request(url: str) -> str:
         raise PipxError(str(e)) from e
 
 
+PEP723 = re.compile(
+    r"(?m)^# /// (?P<type>[a-zA-Z0-9-]+)$\s(?P<content>(^#(| .*)$\s)+)^# ///$"
+)
+
+
 def _get_requirements_from_script(content: str) -> Optional[List[str]]:
-    # An iterator over the lines in the script. We will
-    # read through this in sections, so it needs to be an
-    # iterator, not just a list.
-    lines = iter(content.splitlines())
-
-    for line in lines:
-        if not line.startswith("#"):
-            continue
-        line_content = line[1:].strip()
-        if line_content == "Requirements:":
-            break
-    else:
-        # No "Requirements:" line in the file
+    """
+    Supports PEP 723.
+    """
+
+    name = "pyproject"
+
+    # Windows is currently getting un-normalized line endings, so normalize
+    content = content.replace("\r\n", "\n")
```

Comment:
Is this needed? From what I can tell not normalising would simply produce some harmless empty lines that do not match anyway.

Reply:
(The CI was breaking on Windows otherwise ;) )